src/InterpLib/Error.ml

```diff
@@ -29,12 +29,12 @@ let report ?pos ~cls msg =
     | Warning -> "warning"
     | Note    -> "note"
   in
-  match pos with
+  match Option.bind pos Position.get_text_range with
```

Comment:
I think we should not use `Option.bind` here, because we have three cases instead of two:

1. we do not want to print the position (`pos = None`)
2. we want to print the position, but we are unable to extract the text range, e.g. in REPL, but I thing it would change in the future; other example is when line directives (issue #128) are broken
3. we get the text range that we can print.

Current implementation merges the first two cases.